src/core/api/option_parsers.ts

```diff
@@ -511,7 +515,10 @@ function parseLoadVideoOptions(
 
   if (!isNullOrUndefined(options.url)) {
     url = String(options.url);
-  } else if (isNullOrUndefined(options.transportOptions?.manifestLoader)) {
+  } else if (
+    isNullOrUndefined(options.transportOptions?.initialManifest) &&
+    isNullOrUndefined(options.transportOptions?.manifestLoader)
+  ) {
     throw new Error("No url set on loadVideo");
```

Comment:
Maybe it would be better here to explain that no url AND manifest loader / initial manifest were provided.

Reply:
OK

Reply:
Done